tasks.py

```diff
@@ -118,22 +136,6 @@ def deps(ctx: Context) -> None:
     ctx.run(" ".join(cmds), echo=True, pty=True)
 
 
-@invoke.task
-def build(ctx: Context) -> None:
-    """Build the GX Agent Image"""
-    cmds = [
-        "docker",
-        "buildx",
-        "build",
-        "-t",
-        "greatexpectations/agent",
-        "-f",
-        DOCKERFILE_PATH,
-        ".",
-    ]
-    ctx.run(" ".join(cmds), echo=True, pty=True)
```

Comment:
This seems like a duplicate of the `docker` task.
What's the difference between `buildx build` and `build`?

Reply:
https://docs.docker.com/build/architecture/

Buildx is the new build tool that docker uses. Under the hood newer versions of docker run buildx when you run docker build.